pg/wal/cursor.go

```diff
@@ -51,8 +51,8 @@ func (c Cursor) MoveTo(location Location) Cursor {
 	return Cursor{location, c.reader}
 }
 
-// ReadEntry will read a tuple at the current location and if successful advance to the next tuple
-func (c Cursor) ReadEntry() (entry *Entry, cur Cursor, err error) {
+// ReadEntries will read a tuple at the current location and if successful advance to the next tuple
```

Comment:
The language around that wasn't really correct in the first place.  I'll rephrase.